content/terms/self-hosted.md

```diff
@@ -164,13 +164,13 @@ As used on this page, the following terms have the following specific meanings:
 
 <span className="tw-text-blurple-400">
   ## 3. Acceptable use
-</span>
 
 <span className="tw-text-gray-400">
   <i>**Short version**: You’re allowed to use Sourcegraph for any reasonable purposes (e.g., don’t try to re-sell it, don’t violate your license, etc.). If you post any comments or other content, you’re responsible for it, and we have the right to display it to other users per your settings. If you give us a great idea about how to improve our services, we can use it.</i>
+  
```

Comment:
We should remove any empty lines. Running prettier locally (`yarn prettier`) will fix these issues

```suggestion
```